.github/ISSUE_TEMPLATE/Software-Project-Contribution.md

```diff
@@ -1,65 +1,86 @@
+# Prerequisites
+
+## Membership requirement to contribute new projects
+
+Please note that only FINOS members can propose new projects. If you're interested in membership, see [here](https://www.finos.org/membership-benefits#become-a-member). You can still submit this contribution if your organization is not a [FINOS member](https://finos.org/members), but you will be required to have a maintainer from a FINOS member.
+
+- [ ] I confirm I am contributing on behalf of a FINOS member (`add FINOS member organization name`) or I will seek a maintainer from a FINOS member during socialization 
+
+## Understading the contribution Process
+
+Completing an onboarding of a project into FINOS comprises following 5 main steps described in details in our [Community Repository](https://community.finos.org/docs/governance/Software-Projects/contribution#contribution-of-an-existing-code-base-into-finos-as-a-new-project). 
+
+ - [ ] I confirm I have reviewed the contribution process.
+
+## Understanding FINOS
 
+As a project onboarding into FINOS, you should familiarize yourself and your contributor team with the following FINOS information: [FINOS overview](https://www.finos.org/hubfs/An%20Introduction%20to%20FINOS.pdf), [FINOS Software Projects Governance](/docs/governance/#open-source-software-projects), [FINOS Project Lifecycle](/docs/governance/Software-Projects/project-lifecycle), [FINOS Maintainers cheatsheet](https://community.finos.org/docs/finos-maintainers-cheatsheet/).
 
-# Onboarding Process
+- [ ] I confirm I am familiar with the FINOS governance and maintainers responsibilities
 
-Completing an onboarding of a project into FINOS requires following these 5 main steps:  
+---
+# Contribution information
+
+## Describing The Contribution
 
-1.  [**Describing the Contribution**](#1-describing-the-contribution) _led by contributor_
-2.  [**Approval**](#2-approval) _led by FINOS TOC_
-3.  [**Preparing for Onboarding**](#3-preparing-for-onboarding) _led by contributor_
-4.  [**Onboarding**](#4--finos-onboarding) _completed by FINOS Infra_
-5.  [**Announcement**](#4--finos-onboarding) _led by FINOS Marketing_
+This section contains information to be provided by the contributor to give it the best chances to be approve stage. They largely mirror the requirements described in the [incubation checklist](/docs/governance/software-projects/stages/incubating/#incubating-lifecycle-checklist). Please fill below as appropriate:
 
-# 1. Describing The Contribution
+### Proposed Project Metadata
+- Name: `enter here proposed project name`
+- Slug: `enter here proposed project description`
 
-This is a list of questions that need to be answered by the contributor in order to allow a new project to pass to the approval stage of onboarding.
+> Please review the [trademark checks](docs/governance/Software-Projects/contribution#trademark) that will be performed on the projects.
```

Comment:
Is this a todo? I can't find a trademark header.

Reply:
https://community.finos.org/docs/governance/Software-Projects/contribution#trademark ? 

I think it might be missing a / - I fixed it upstream. Please double check @TheJuanAndOnly99 and resolve